conference.js

```diff
@@ -366,7 +369,8 @@ class ConferenceConnector {
         logger.error('CONFERENCE Error:', err, params);
         switch (err) {
         case ConferenceErrors.CHAT_ERROR:
-            {
+            logger.error("Chat error.", err);
```

Comment:
Did you find it useful to explicitly log the chat error instead of relying on the logger.error call above?

Reply:
We log an unknown error below, so I figured it would be useful to have that one as well...I've never seen this error, so not sure how it looks in the err object..Maybe it's useless to have it explicitly.